packages/aws-cdk/lib/api/cloudformation/nested-stack-helpers.ts

```diff
@@ -5,15 +5,9 @@ import { formatErrorMessage } from '../../util';
 import type { SDK } from '../aws-auth';
 import { LazyListStackResources, type ListStackResources } from './evaluate-cloudformation-template';
 import { CloudFormationStack, type Template } from './stack-helpers';
+import type { NestedStackTemplates } from '../../../../@aws-cdk/tmp-toolkit-helpers/src/api';
 
-export interface NestedStackTemplates {
-  readonly physicalName: string | undefined;
-  readonly deployedTemplate: Template;
-  readonly generatedTemplate: Template;
-  readonly nestedStackTemplates: {
-    [nestedStackLogicalId: string]: NestedStackTemplates;
-  };
-}
+export { NestedStackTemplates } from '../../../../@aws-cdk/tmp-toolkit-helpers/src/api';
```

Comment:
is it reasonable to both `export` and `import` `NestedStackTemplate` here?

Reply:
yes, it's just going to annoy me when I do the move :D